doc/releases/changelog-dev.md

```diff
@@ -272,113 +259,147 @@
   >>> relative_entropy_circuit((x,), (y,))
   0.017750012490703237
   ```
-  
-* New PennyLane-inspired `sketch` and `sketch_dark` styles are now available for
-  drawing circuit diagram graphics. 
-  [(#2709)](https://github.com/PennyLaneAI/pennylane/pull/2709)
 
-* Added `QutritDevice` as an abstract base class for qutrit devices.
-  ([#2781](https://github.com/PennyLaneAI/pennylane/pull/2781), [#2782](https://github.com/PennyLaneAI/pennylane/pull/2782))
-* Added operation `qml.QutritUnitary` for applying user-specified unitary operations on qutrit devices.
-  [(#2699)](https://github.com/PennyLaneAI/pennylane/pull/2699)
+  - Support in `qml.math` for flexible post-processing:
+
+  ```pycon
+  >>> rho = np.array([[0.3, 0], [0, 0.7]])
+  >>> sigma = np.array([[0.5, 0], [0, 0.5]])
+  >>> qml.math.relative_entropy(rho, sigma)
+  tensor(0.08228288, requires_grad=True)
+  ```
+
+<h4>Backpropagation with Jax for DefaultMixed 🙌</h4>
 
-* Added `default.qutrit` plugin for pure state simulation of qutrits. Currently supports operation `qml.QutritUnitary` and measurements `qml.state()`, `qml.probs()`.
-  [(#2783)](https://github.com/PennyLaneAI/pennylane/pull/2783)
+* The `default.mixed` device now supports [backpropagation](https://pennylane.readthedocs.io/en/stable/introduction/unsupported_gradients.html#backpropagation) with the `"jax"` interface.
+  [(#2754)](https://github.com/PennyLaneAI/pennylane/pull/2754)
+
+  ```python
+  dev = qml.device("default.mixed", wires=2)
+
+  @qml.qnode(dev, diff_method="backprop", interface="jax")
+  def circuit(angles):
+      qml.RX(angles[0], wires=0)
+      qml.RY(angles[1], wires=1)
+      return qml.expval(qml.PauliZ(0) + qml.PauliZ(1))
+  ```
 
   ```pycon
-  >>> dev = qml.device("default.qutrit", wires=1)
-  >>> @qml.qnode(dev)
-  ... def circuit(U):
-  ...     qml.QutritUnitary(U, wires=0)
-  ...     return qml.probs(wires=0)
-  >>> U = np.array([[1, 1, 0], [1, -1, 0], [0, 0, np.sqrt(2)]]) / np.sqrt(2)
-  >>> print(circuit(U))
-  [0.5 0.5 0. ]
+  >>> angles = np.array([np.pi/6, np.pi/5], requires_grad=True)
+  >>> qml.grad(circuit)(params)
+  array([-0.8660254 , -0.25881905])
   ```
+
+<h4>More support for Jax and Tensorflow tensors 😻</h4>
+
+* Quantum channels now support Jax and Tensorflow tensors.
+  [(#2776)](https://github.com/PennyLaneAI/pennylane/pull/2776)
   
-  
-**Operator Arithmetic:**
+  This allows quantum channels to be used inside QNodes decorated by `tf.function`, 
+  `jax.jit`, or `jax.vmap`:
```

Comment:
I wonder if we need more examples here, or if the example needs to be more in-line with the copy? The code example just uses the Jax interface, not actually jit'ing the QNode. Thoughts?

Reply:
No here it is sufficient the UI is the same for jax jit and jax vmap. We should be concise as well.